common/src/main/java/org/dash/wallet/common/data/entity/GiftCard.kt

```diff
@@ -21,10 +21,11 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 import com.google.zxing.BarcodeFormat
 import org.bitcoinj.core.Sha256Hash
-
+// TODO: need migration for 12->13 for addition of id column?
 @Entity(tableName = "gift_cards")
 data class GiftCard(
     @PrimaryKey var txId: Sha256Hash,
+    var id: String? = "",
```

Comment:
Because the gift card table is in production - yes, we will need a migration.
(This TODO might not be needed as it looks like the migration is added already).